src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,401 @@
+            self._check_stop()
+            self.wait_click_ocr(
+                box=box_cancel,
+                match=re.compile(r"取消"),
+                time_out=5,
+                after_sleep=0.5,
+                target_height=720
+            )
+            self.sleep(1.0)
+            self.log_info("低保金领取流程完成")
+
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)